doc/modules/mod_pubsub.md

```diff
@@ -0,0 +1,67 @@
+### What is PubSub?
+
+PubSub is a fascinating design pattern which mostly promotes a loose coupling between two kinds of entity (publishers and subscribers).
+Like their names suggest, in the pubsub world we have publishers who fire events, and subscribers who wish to be notified about those events when publishers push data.
```

Comment:
Subscribers doesn't have to be online, in this case they can still get the "unread" events from the node if they wish to.